OpenRA.Game/Exts.cs

```diff
@@ -375,6 +375,11 @@ public static IEnumerable<T> Append<T>(this IEnumerable<T> ts, params T[] moreTs
 			return ts.Concat(moreTs);
 		}
 
+		public static IEnumerable<T> Exclude<T>(this IEnumerable<T> ts, params T[] exclusions)
```

Comment:
Is the idea of `Exclude` instead of `Except` to avoid potential future namespace collision?

Reply:
Not nearly as much thought as that. I just wanted a method like `Append` above that took a params array. Since `Append` was named differently than it's underlying `Concat`, I choose a different name here too.

Happy to rename to whatever though.